setup.cfg

```
@@ -56,6 +56,9 @@ tests =
     pytest-rerunfailures
 ipython =
     ipython
+extra =
```

```suggestion
extras =
```
This is grammatically slightly more correct and is more consistent with `graphics` and `tests`.